website/docs/07-apps/03-network-features/07-vnc-server.md

```diff
@@ -3,27 +3,27 @@ slug: /network/vnc
 ---
 
 
-# VNC server
+# VNC Server
 
 ![](./assets/vnc.webp)
 
-Remote display and remote control of your Miyoo Mini Plus from your computer thanks to VNC.
+Remote display and remote control of your Miyoo Mini+ from your computer thanks to VNC.
 
 
 ## Features
 
-- Remote display of the Miyoo Mini Plus
-- Remote control of the Miyoo Mini Plus
+- Remote display of the Miyoo Mini+
+- Remote control of the Miyoo Mini+
 
 
 ## Enabling VNC access
 
 You'll find it in Tweaks app: [`Network` -› `VNC: Screen share`](/docs/apps/tweaks#vnc-screen-share).
 
-You can either use the master toggle to turn the VNC server on with your Right/Left arrows, or press A to enter the submenu and you'll get 2 options:
+You can either use the master toggle to turn the VNC Server on with your right/left arrows, or press A to enter the submenu and you'll get 2 options:
 
 1. **Enable** - Activate VNC Server
-2. **Framerate** - Set the framerate of the VNC Server between 1 and 20. The higher the framerate, the more CPU will be use.
+2. **Framerate** - Set the framerate of the VNC Server between 1 and 20. The higher the framerate, the more CPU will be use
```

Comment:
You ok to add the "d" on the end of "use" (used) then this one's good to go - Thanks!

Reply:
Added on main PR. I close this one. Thanks!